contracts/Core/parameters/Governance.sol

```diff
@@ -106,6 +106,7 @@ contract Governance is Initializable, ACL, Constants {
     }
 
     function setMaxAge(uint32 _maxAge) external initialized onlyRole(GOVERNER_ROLE) {
+        require(_maxAge <= 100 * 10000, "Invalid Max Age Update");
```

Comment:
Does it make sense to assign this to say `maxSafeAge`? We may end up forgetting about this require if we change the age mapping.